services/web/server/src/simcore_service_webserver/projects/_folders_api.py

```diff
@@ -0,0 +1,68 @@
+import logging
+
+from aiohttp import web
+from aiopg.sa.engine import Engine
+from models_library.folders import FolderID
+from models_library.products import ProductName
+from models_library.projects import ProjectID
+from models_library.users import UserID
+from simcore_postgres_database import utils_folders as folders_db
+from simcore_service_webserver.projects.models import UserProjectAccessRights
+
+from .._constants import APP_DB_ENGINE_KEY
+from ..users.api import get_user
+from .db import APP_PROJECT_DBAPI, ProjectDBAPI
+from .exceptions import ProjectInvalidRightsError
+
+_logger = logging.getLogger(__name__)
+
+
+async def replace_project_folder(
```

Comment:
maybe call this one move as well? since that's what the operation is? replacing a folder is not intuitive for me